content/general/pubsub.textile

```diff
@@ -0,0 +1,21 @@
+---
+title: Pub/Sub
+section: general
+index: 23
+---
+
+Pub/Sub is a shorthand for the Publish/Subscribe architectural pattern, which is the most popularly used pattern in realtime data delivery. This messaging pattern lets any number of publishers publish data, ideally to a data channel/topic, which could be subscribed to by any number of subscribers. The important thing to note about Pub/Sub is that once subscribed, the subscribers no longer have to poll the server or data provider to check if there’s any new data that they need to be aware of, instead, they will be notified of it, as it becomes available.
+
+<a href="/images/diagrams/pub-sub.png" target="_blank">
+  <img src="/images/diagrams/pub-sub.png" style="width: 100%" alt="Pub/Sub in Ably">
+</a>
+
+For instance, consider a location tracking application. The subscriber subscribes to location updates that are being published by the vehicle (a.k.a the GPS tracker within the vehicle). Until the subscriber unsubscribes from these updates, the vehicle continues to send updates to the subscriber as it is moving, thus generating new location data.
+
+<a href="/images/diagrams/ably-pubsub.png" target="_blank">
+  <img src="/images/diagrams/ably-pubsub.png" style="width: 100%" alt="Pub/Sub in Ably">
+</a>
+
+As seen in the illustration above, you can implement Pub/Sub easily by using Ably as the intermediary realtime messaging platform. Your publishers and subscribers can attach to named "channels":/realtime/channels-messages and Ably owns the responsibility of routing the right message to the right client in realtime, with "nominal latencies":https://support.ably.io/support/solutions/articles/3000044625-round-trip-latency-and-performance.
```

Comment:
Instead of "owns the repsonsiblity", perhaps just "Is responsible for routing..."

Reply:
If something is done in real time, it's two words, please review the tone of voice guide.

Reply:
I don't like "nominal latencies", perhaps instead ".. routing messages to subscribed clients in real time (typically within 60ms globally)."